Gruntfile.js

```diff
@@ -1462,6 +1462,17 @@ module.exports = function(grunt) {
 		);
 	} );
 
+	grunt.registerTask( 'copy:theme-json', 'Copies theme.json file contents to theme-json.php.', function() {
+		[ 'theme.json', 'theme-i18n.json' ].forEach( function( fileName ) {
+			var themeData = grunt.file.readJSON(SOURCE_DIR + 'wp-includes/' + fileName);
```

Comment:
Should we formally check if both JSON file are present before the `readJSON` operation? It shouldn't make much difference as we know files will be always present. 

Also, should we add a verification task to check if the files were created. ref (verify:old-files). 
```
	grunt.registerTask( 'verify:old-files', function() {
		const file = `${ BUILD_DIR }wp-admin/includes/update-core.php`;

		assert(
			fs.existsSync( file ),
			'The build/wp-admin/includes/update-core.php file does not exist.'
		);
```